console/lib/address_ctrl.js

```diff
@@ -113,11 +117,11 @@ function create_address(address) {
             }
         ]
     };
-    return request(addr_path, 'POST', {'content-type': 'application/json'}, JSON.stringify(data));
+    return this.request(this.addr_path, 'POST', {'content-type': 'application/json'}, JSON.stringify(data));
 }
 
-function delete_address(address) {
-    return request(addr_path + address.address, 'DELETE');
+AddressCtrl.prototype.delete_address = function(address) {
+    return this.request(this.addr_path + address.address, 'DELETE');
```

Comment:
Minor: Arguably the path needn't be passed in, since all current uses use the instance member variable.

Reply:
Leaving as is since this.addr_path is not used by get_address_types(). For delete_address, the address would have to be appended to the path, but not for create_address. So there's basically 3 different uses of it.